opensrp-chw-core/src/test/java/org/smartregister/chw/core/utils/UtilsTest.java

```diff
@@ -277,6 +278,14 @@ public void testGetDuration() {
         Assert.assertEquals("15y 1m", duration);
     }
 
+    @Test
+    public void testGetStringEntity(){
+        Assert.assertEquals("Events",Utils.getSyncEntityString(SyncEntity.EVENTS));
+        Assert.assertEquals("Locations",Utils.getSyncEntityString(SyncEntity.LOCATIONS));
+        Assert.assertEquals("Plans",Utils.getSyncEntityString(SyncEntity.PLANS));
+        Assert.assertEquals("Tasks",Utils.getSyncEntityString(SyncEntity.TASKS));
+        Assert.assertEquals("",Utils.getSyncEntityString(SyncEntity.STRUCTURES));
+    }
```

Comment:
Why is this returning an empty string ?

Reply:
I tried using the ArguementMatchers.anyString() but it was failing. It is empty because it is expected to return an empty string other than the ones available in the context

Reply:
if the SyncEntity enum passed as an arguement is not there, the default case returns an empty string